py3status/modules/backlight.py

```diff
@@ -81,18 +98,22 @@ def post_config_hook(self):
             raise Exception(STRING_NOT_AVAILABLE)
 
         # check for an error code and an output
-        self.xbacklight = False
+        self.backlight_command_available = False
         try:
-            if self.py3.command_output(['xbacklight', '-get']):
-                self.xbacklight = True
+            output = self.py3.command_output(_command_get())
+            try:
+                float(output)
+                self.backlight_command_available = True
+            except ValueError:
+                self.backlight_command_available = False
         except:
             pass
 
-        if self.xbacklight and self.brightness_initial:
+        if self.backlight_command_available and self.brightness_initial:
             self._set_backlight_level(self.brightness_initial)
 
     def on_click(self, event):
```

Comment:
I'm only using `light` right now... Scrolling here does not change values for me?

Reply:
It works for me now... Perhaps this was the issue? https://github.com/ultrabug/py3status/pull/1077/commits/5cee59a27b38f311743e4c8cc51518d0516effc1#diff-7d7c9df691aab6e0ddfbacd9e0c81b94R133

I'm not super familiar with python